### ISSUE TITLE
load PSReadLine get error when launch developer powershell from Visual Studio

### ISSUE THREAD

**Issue by roachsinai** [PowerShell/PSReadLine#3117]
### Prerequisites

- [X] Write a descriptive title.

### Issue summary

Get below error when trigger powershell in Visual studio:

![image](https://user-images.githubusercontent.com/9500049/146645332-cb0bc90d-16be-4168-b711-3c7fec62563a.png)

```
Set-PSReadLineOption : 找不到与参数名称“PredictionSource”匹配的参数。                                                   所在位置 C:\Users\xq\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1:10 字符: 22                           + Set-PSReadLineOption -PredictionSource History                                                                        +                      ~~~~~~~~~~~~~~~~~                                                                                    + CategoryInfo          : InvalidArgument: (:) [Set-PSReadLineOption]，ParameterBindingException                        + FullyQualifiedErrorId : NamedParameterNotFound,Microsoft.PowerShell.SetPSReadLineOption                                                                                                                                                   Set-PSReadLineOption : 'InlinePrediction' 不是有效的颜色值                                                              所在位置 C:\Users\xq\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1:30 字符: 1                            + Set-PSReadLineOption -Colors @{ InlinePrediction = '#000044'}                                                         + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                                             + CategoryInfo          : NotSpecified: (:) [Set-PSReadLineOption], ArgumentException                                   + FullyQualifiedErrorId : System.ArgumentException,Microsoft.PowerShell.SetPSReadLineOption                                                                                                                                                 **********************************************************************                                                  ** Visual Studio 2019 Developer PowerShell v16.11.7                                                                     ** Copyright (c) 2021 Microsoft Corporation                                                                             **********************************************************************
```

which means settings:

```
Set-PSReadLineOption -PredictionSource History
Set-PSReadLineOption -Colors @{InlinePrediction = '#000044'}
```

are not support.

But no error reports when  I launch powershell in Windows Terminal. Is there a way to fix these errors?

Thanks in advance.

**Comment by daxian-dbw** [comment]
Please check the version of the PSReadLine used in your "developer powershell from Visual Studio". The predictive intellisense feature was added in 2.1 version of PSReadLine. You may need to update the PSReadLine module.

**Comment by roachsinai** [comment]
> Please check the version of the PSReadLine used in your "developer powershell from Visual Studio". The predictive intellisense feature was added in 2.1 version of PSReadLine. You may need to update the PSReadLine module.

Thanks for your reply!

In developer powershell the version of PSReadLine is 2.0.0, and the RSReadLine version in Regular PowerShell is 2.2.0.

So, maybe I show run `Install-Module PSReadLine -AllowPrerelease -Force` in developer powershell?

**Comment by daxian-dbw** [comment]
Yes, please upgrade the PSReadLine module, check out https://github.com/PowerShell/PSReadLine#upgrading for instructions.

**Comment by roachsinai** [comment]
Close all powershell instance and run `powershell -noprofile -command "Update-Module PSReadLine -AllowPrerelease"` not work for me, still get above error.

The problem for me now is I've installed PSReadLine 2.2.0, but Visual Studio developer powershell use PSReadLine Module 2.0.0 version.

Is there a way to fix it?

Use `Get-Module -ListAvailable` I found 2.2.0 was installed at ` C:\Users\$env:USERNAME\Documents\WindowsPowerShell\Modules` and 2.0.0 was located at `C:\Program Files\WindowsPowerShell\Modules`
